websites/documentation/src/content/writing/images.mdx

```diff
@@ -1,17 +1,67 @@
 ---
-title: Images
+title: Images and Diagrams
 wideLayout: true
 ---
 
-You can find all images in `/src/images`, and can use standard Markdown syntax to insert images. It is good to have a meaningful image names and alt text when using images. A meaningful alt text is useful for visually impaired users using softwares to read text on a page. Also, a meaningful image name with keywords helps with SEO.
+# Images
```

Comment:
I chose to document the diagram feature on the same page like images and wrote a more proper documentation for images on that occasion.  It was pretty poor.